rest_framework/fields.py

```diff
@@ -989,9 +987,9 @@ def __init__(self, max_digits, decimal_places, coerce_to_string=None, max_value=
         self.min_value = min_value
 
         if self.max_value is not None and not isinstance(self.max_value, decimal.Decimal):
-            logger.warning("max_value in DecimalField should be Decimal type.")
+            warnings.warn("max_value should be a Decimal instance.")
         if self.min_value is not None and not isinstance(self.min_value, decimal.Decimal):
-            logger.warning("min_value in DecimalField should be Decimal type.")
+            warnings.warn("min_value should be a Decimal instance.")
```

Comment:
Could you add a `stacklevel` param so that python displays a trace with the faulty calling code rather than this line?